src/main/java/edu/kit/kastel/sdq/artemis4j/grading/metajson/AnnotationDTO.java

```diff
@@ -14,7 +14,9 @@ public record AnnotationDTO(
         @JsonProperty String uuid,
```

Comment:
I'd suggest to use the records like LineColumn directly.

Reply:
I think this would break compatibility with existing submissions?

Reply:
Yes. But this is no problem as we are currently between semesters :)